frontend/src/metabase/visualizations/shared/utils/sizes.ts

```diff
@@ -43,23 +43,23 @@ const VISUALIZATION_SIZES: {
   },
   pie: {
     min: { width: 4, height: 3 },
-    default: { width: 8, height: 6 },
+    default: { width: 12, height: 8 },
   },
   funnel: {
     min: { width: 4, height: 3 },
     default: { width: 8, height: 6 },
```

Comment:
Any reason why funnels are still 8 default? That's the only one here that I was wondering, why not bump that up to 12 as well.

Reply:
Nope, just forgot. Thanks for the reminder.